src/client/lazy-app/Modal/ModalHint/style.css

```diff
@@ -0,0 +1,31 @@
+span.modal-hint {
```

Comment:
Why require a particular kind of element here? (`span`)

Reply:
No reason in particular, just seemed to make it easier to understand

Reply:
The good thing about using class names (and CSS modules) is that you can decouple style and semantics. So, if a particular `modal-hint` made sense as an `<aside>`, then that's fine, because the class name can be applied to any element. However, in this case it can only be applied to `<span>`, and I don't think that's a useful restriction.

In general I'd just use class names, and only use tag names in cases where it's justified. Cases like `.icon > svg` are fiiiine, and your use of tag names for dialog content is great, but I think just sticking with a class name is better here.